OpenRA.Game/Sync.cs

```diff
@@ -194,7 +194,7 @@ public static T RunUnsynced<T>(bool checkSyncHash, World world, Func<T> fn)
 			finally
 			{
 				inUnsyncedCode = false;
-				if (sync != world.SyncHash())
+				if (!world.Disposing && sync != world.SyncHash())
```

Comment:
Can we please include a comment here explaining that the hash changes to 0 when disposing / whatever you think is appropriate?

Reply:
Added a comment.